### ISSUE TITLE
Support idrac8

### ISSUE THREAD

**Issue by arozenfe** [rebootoio/vaxiin-server#19]
Currently you only support idrac9. Is it possible to add support for idrac8 as well?

Thanks!

**Comment by balexx** [comment]
Yes! We need hardware to develop and test on, though. Is that something you could help with?

**Comment by arozenfe** [comment]
We have this hardware, but I cannot provide access to it, it's inside our DC. Is there any way I could assist? Maybe by providing the html code and inspections of different pages and frames? I could test it on our machines.

**Comment by em-winterschon** [comment]
I can also assist in testing idrac8 (and idrac7) based systems, though they are not public so I cannot provide remote access. However, I'm happy to run any scripts or debug commands with results posted here asynchronously.

**Comment by balexx** [comment]
@em-winterschon Great! could you pop onto [our Discord server](https://discord.gg/aEJ6qwcCGs) under #support so we could discuss the details?
